README.md

```diff
@@ -0,0 +1,87 @@
+## CEL Policy
+
+This policy is a meta-policy that allows to run [CEL](https://github.com/google/cel-go) expressions
+against Kubernetes resources.
+A meta-policy is a policy that can be configured via settings, and does not require to be recompiled to change its behavior, acting as a DSL.
+
+The settings of the policy are compliant with the [ValidatingAdmissionPolicy Kubernetes resource](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/),
```

Comment:
maybe worth to say that is implemented using apimachinery's CEL library in a throwaway comment.

Reply:
it is not implemented through apimachinery. The original (apiserver) code is not imported by the policy. The only thing we use are the extensions, we are using the vanilla cel-go library.

Reply:
good point, i will add the clarification

Reply:
added in https://github.com/kubewarden/cel-policy/pull/5/commits/1b1afdef1a87b351af228089d5198c5140c5ad49